registry/polkadot.json

```
@@ -413,4 +413,12 @@
         "longDescription": "Pirate navigating the seas of OpenGov 🏴‍☠️—voting on cool stuff whenever I can. I run the largest Polkadot content portal & ecosystem wiki too - https://dablock.com",
         "isOrganization": false
     }
+    {
+    "address": "14fp9FUSaeRpLJ8i7pDmNy7zVZNiZnNmDJ1bHEaZpXPqb1AX",
+    "name": "treebtc", 
+    "image": "https://raw.githubusercontent.com/asggWa/opengov-delegate-registry/refs/heads/master/images/20240923_083455_0000.png"    ,
```

also, please, replace the path to novasamatech repo
```suggestion
    "image": "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/20240923_083455_0000.png",
```

---

did it